### PR TITLE
💄 Change status text on case cards

### PR DESCRIPTION
**Pull request by Sven65**
## Explain the changes you’ve made

In accordance with #CU-1nwu0yq, I've changed the text on action cards to "Komplettera ansökan" when a completion is required, such as status `active:completion:required`.

## Explain your solution

I've added a check for the status type that changes the text

## How to test

Concrete example:

1. Checkout this branch
2. Get a case with the status
3. See that the text of the button is "Komplettera ansökan"

## Tested environments

- [] Storybook on a iOS device/simulator.
- [x] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

---

## Checklist for reviewers

Here are a few points that are useful for reviewers to check.

The short list of the (maybe) most important points are:

- Does this PR work as expected?
- Does the PR properly only concern a single feature/bug/thing?
- Is the code clear and concise?
- Will the changes impact existing functionality?
- Are there any linting errors (ESLint) that should be resolved?
- Do the automatic GitHub checks pass?

However it is good if a reviewer can be as thorough as possible,
within reasonable limits of course.

### Functionality

- Does this PR work as expected?
- If this is a new feature, does it add value, or is it a sign of feature-creep?
- Will the changes impact existing functionality?
- Does the changes create inconsistencies somewhere else?
- Does the code work on different platforms if applicable?
- How does the code handle if unexpected values are passed to functions etc.?

### Readability, syntax, formatting, style

- Is the code clear and concise?
- Has styling rules (Prettier) been applied to included files?
- Are functions/variables/etc. aptly named?

### Design

- Is the code properly planned and designed?
- Is code organized/separated in a logical fashion?
- Does the code follow clean code practices to a reasonable degree?
- Is the code self-documenting enough?
- Is the code in sync with existing patterns/technologies?
- Does the code reuse existing functionality where applicable?
- Is the code designed with reusability in mind?
- Does the code have hard-coded values that can be refactored?

### Patterns, idioms, tests, best practices

- Are there any linting errors (ESLint) that should be resolved?
- Is the code properly typed (TypeScript)?
- Does the code utilize language features to their full potential?

### Debugging, testing, configuring

- Is appropriate logging used?
- Have tests been added/modified to accommodate the changes?
- Is the feature sufficiently configurable in case tweaks are required in the future?

### Performance, reliability, scalability

- Is the code well optimized?
- Does the code scale?
- Does the code handle failure scenarios?

### Security

- Is there any obvious potential for exploits and security threats?
- Does code check proper authorization where applicable?

### Etiquettes

- Are commits atomic (does only one thing per commit)?
- Does the PR properly only concern a single feature/bug/thing?
- Are commit messages well-written?
- Do the automatic GitHub checks pass?

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1nwu0yq Uppdatera text på action-knapp på ärendekort](https://app.clickup.com/t/1nwu0yq)

**Comment by sebastianthulin** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/helsingborg-stad/repositories/fedfce3fd8695f238dea9cb00bd37ce7/pull-request/601)

File: source/screens/caseScreens/CaseOverview.tsx [hunk 1]
```diff
@@ -142,7 +142,9 @@ const computeCaseCardComponent = (caseData, navigation, authContext, extra) => {
   const statusType = caseData?.status?.type || "";
   const isNotStarted = statusType.includes("notStarted");
   const isOngoing = statusType.includes("ongoing");
-  const isCompletionRequired = statusType.includes("completionRequired");
+  const isCompletionRequired =
+    statusType.includes("completionRequired") ||
+    statusType.includes("active:completion:required");
```
Comment (dannil76) [inline source/screens/caseScreens/CaseOverview.tsx:147]: There is now two kind of completion status that a case can have

'Ansökan behöver kompletteras' = active:completionRequired:viva'
'Stickprovskontroll' = active:completionRandomCheckRequired:viva

I think you need to update the code to make the use of them in a correct manner. :)

**Review by dannil76 — CHANGES_REQUESTED** [review]

File: source/screens/caseScreens/CaseOverview.tsx (hunk 1 above)
Comment (Sven65) [reply source/screens/caseScreens/CaseOverview.tsx:147]: Can it still have the standard `completionRequired` status? The task only lists `active:completion:required` as the status to be added.

Comment (karlknowit) [reply source/screens/caseScreens/CaseOverview.tsx:147]: We might need to align with each otherhere @Sven65

**Comment by karlknowit** [comment]
This PR might be redundant in favour for
https://github.com/helsingborg-stad/app-mitt-helsingborg/pull/602/files

File: source/screens/caseScreens/CaseOverview.tsx (hunk 1 above)
Comment (karlknowit) [reply source/screens/caseScreens/CaseOverview.tsx:147]: @Sven65 Dont think that this PR is needed, you can close this one.

**Pull request closed without merging**